deploy/infra/create_cpu_batch_account_pool.bicep

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+param environmentCode string
+param projectName string
+
+// Name parameters for infrastructure resources
+param batchAccountName string
+param batchAccountResourceGroup string
+param batchAccountPoolCheckUamiName string
+param batchAccountLocation string
+
+// Mount options
+param batchAccountPoolMountAccountName string
+param batchAccountPoolMountAccountKey string
+param batchAccountPoolMountFileUrl string
+
+// Parameters with default values  for Data Fetch Batch Account Pool
+param batchAccountCpuOnlyPoolName string = 'data-cpu-pool'
+param batchAccountCpuOnlyPoolVmSize string = 'standard_d2s_v3'
+param batchAccountCpuOnlyPoolDedicatedNodes int = 1
+param batchAccountCpuOnlyPoolImageReferencePublisher string = 'microsoft-azure-batch'
+param batchAccountCpuOnlyPoolImageReferenceOffer string = 'ubuntu-server-container'
+param batchAccountCpuOnlyPoolImageReferenceSku string = '20-04-lts'
+param batchAccountCpuOnlyPoolImageReferenceVersion string = 'latest'
+param batchAccountCpuOnlyPoolStartTaskCommandLine string = '/bin/bash -c "apt-get update && apt-get install -y python3-pip && pip install requests && pip install azure-storage-blob && pip install pandas"'
```

Comment:
This is not required. Sorry for the confusion. There is no requirement for startup command to install packages.

Reply:
pool-name in the script accepts a user input and if user is not providing any input for the name then it is defaulted to `${environment_code}-data-cpu-pool`

Reply:
Cleaned